copier/template.py

```diff
@@ -88,13 +88,17 @@ def load_template_config(conf_path: Path, quiet: bool = False) -> AnyByStrDict:
     Raises:
         InvalidConfigFileError: When the file is formatted badly.
     """
+
+    class _Loader(yaml.FullLoader):
+        ...
```

Comment:
Would you please add a docstring here explaining why this class exists?

```suggestion
    class _Loader(yaml.FullLoader):
        """Intermediate class to avoid monkey-patching main loader."""
```

Reply:
Makes sense, done. Thanks! 🙏